code/modules/clothing/masks/gasmask.dm

```diff
@@ -109,35 +109,51 @@
 
 /obj/item/clothing/mask/gas/clown_hat
 	name = "clown wig and mask"
-	desc = "A true prankster's facial attire. A clown is incomplete without his wig and mask."
+	desc = "A true prankster's facial attire. A clown is incomplete without his wig and mask. Its form can be changed by using it in your hand."
 	icon_state = "clown"
 	item_state = "clown_hat"
 	flags = BLOCK_GAS_SMOKE_EFFECT | AIRTIGHT | BLOCKHAIR
 	resistance_flags = FLAMMABLE
 	dog_fashion = /datum/dog_fashion/head/clown
 
-/obj/item/clothing/mask/gas/clown_hat/attack_self(mob/user)
-
+/obj/item/clothing/mask/gas/clown_hat/attack_self(mob/living/user)
 	var/mob/M = usr
```

Comment:
this line needs to go, instead of using `M` and `usr` here, it should just use the provided `mob/living/user` var